cmds/cpio/cpio.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// cpio operates on cpio files using a cpio package
+// It only implements basic cpio options.
+//
+//
+// Synopsis:
+//     cpio
+//
+// Description:
+//
+// Options:
+//     o: output an archive to stdout given a pattern
+//     i: output files from a stdin stream
+//     -d: debug prints
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+)
+
+var (
+	debug = func(string, ...interface{}) {}
+	d     = flag.Bool("d", false, "Debug prints")
+)
+
+func usage() {
+	log.Fatalf("Usage: cpio")
```

Comment:
The `flag` package has default flags for `-h` and `-help` which call `flag.Usage` and exit, so overriding the function will print the same usage message using `-h` or with invalid arguments:

```
flag.Usage = func () {
    log.Fatalf("Usage: cpio")
}
```

Don't worry about fixing it now. If I have time later this week, I can go through each command and rewrite them this way.